src/main/webapp/app/exercises/programming/manage/update/update-components/custom-build-plans/programming-exercise-custom-aeolus-build-plan.component.ts

```diff
@@ -211,4 +200,6 @@ export class ProgrammingExerciseCustomAeolusBuildPlanComponent implements OnChan
         }
         this.programmingExercise.windFile.metadata.docker.image = dockerImage.trim();
     }
+
+    protected readonly MarkdownEditorHeight = MarkdownEditorHeight;
```

Comment:
Can you move it to the top alongside the other variables?

Reply:
I just noticed that it's unused 😅 I removed it